ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -58,24 +58,14 @@ chain:
       kubevirt)
           export TEST_PROVIDER='{"type":"kubevirt"}'
 
-          # These are known tests that will not pass with the KubeVirt Platform right now
-          # Resolving these test skips are all being tracked in this epic, https://issues.redhat.com/browse/CNV-30259
-          #
-          # https://issues.redhat.com/browse/CNV-30268
-          if [[ -z "$TEST_SKIPS" ]]; then
-              TEST_SKIPS='should serve endpoints on same port and different protocol for internal traffic on Type LoadBalancer'
-          else
-              TEST_SKIPS=$TEST_SKIPS'\|should serve endpoints on same port and different protocol for internal traffic on Type LoadBalancer'
-          fi
```

Comment:
This is is great. However, since now we are not always setting `TEST_SKIPS`, then in line 73 we get the the redundant pipe `|` as a prefix to `TEST_SKIPS`.

Do you want to copy this logic to line 73?

Reply:
thanks, now if `$TEST_SKIPS` is empty, there will be no pipe.